### PR TITLE
Add fixed cookie dependency for Android 12+

### PR DESCRIPTION
**Pull request by abnegate**
Add fixed cookie store dependency to resolve cookies issues on Android 12+

### CODE REVIEW

**Review by lohanidamodar — APPROVED** [review]

File: src/SDK/Language/Android.php [hunk 1]
```diff
@@ -47,207 +44,193 @@ public function getFiles()
...
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/cookies/InMemoryCookieStore.kt',
+                'template'      => '/android/library/src/main/java/io/appwrite/cookies/InMemoryCookieStore.kt.twig',
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/library/src/main/java/{{ sdk.namespace | caseSlash }}/cookies/SharedPreferencesCookieStore.kt',
+                'template'      => '/android/library/src/main/java/io/appwrite/cookies/SharedPreferencesCookieStore.kt.twig',
```
Comment (christyjacob4) [inline src/SDK/Language/Android.php:182]: Do you think a better naming convention is to have

io/appwrite/cookie
  - store
      - Extensions.kt
      - InMemory.kt
      - SharedPreference.kt 
  - Internal.kt 


WDYT @abnegate  ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Looks good 👍 
Just a minor comment about the folder and file structure ...

File: src/SDK/Language/Android.php (hunk 1 above)
Comment (abnegate) [reply src/SDK/Language/Android.php]: @christyjacob4 I think the folder structure provides a better separation of concerns but the naming makes it hard to tell what the classes are doing without reading some code.

I have updated with just the directory changes, does that work for you?

Comment (christyjacob4) [reply src/SDK/Language/Android.php]: Understood and your change makes sense 👍  Thank you

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**